services/mysql.rst

```diff
@@ -11,7 +11,7 @@ To use it in your application, add it to your ``.symfony/services.yaml``:
     # .symfony/services.yaml
     mydatabase:
         # supported versions: 5.5, 10.0, 10.1, 10.2
-        # 5.5 is also available but not maintained upstream
+        # 5.4 is also available but not maintained upstream
```

Comment:
`5.4` is not available, `5.5` is the available but not maintained version and should be removed from the supported versions list instead.

Reply:
also, the support version list misses `10.3` and `10.4` if you want to add them at the same time.

Reply:
Oops, merged too fast, fixed in 1ce4fcb

Reply:
thanks 👍